kim_cole/Assignments/CoinTosses.py

```diff
@@ -0,0 +1,22 @@
+import random
+
+def tosses():
+	print "Starting the program"
+	trial = 1
```

Comment:
you actually don't need to track a trial variable.  You're already tracking the variable i in range so you can use that.  You would just have to start your range at 1 instead of 0.